webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -268,6 +268,7 @@ class ManageManifestModal extends Component {
                   rows={manifestHistory.results}
                   columns={columns}
                   emptyState={emptyStateData()}
+                  rowKey="created"
```

Comment:
Is this guaranteed to be unique?

Reply:
good question, I can double check how granular the timestamp is

Reply:
The timestamp looks like its down to the second, here is the an example of the actual key react uses `2019-01-29T16:28:59+0000-row`

The object we are getting looks like this: 
```javascript
{
"statusMessage":"test_manifest_roles file imported successfully.","
status":"SUCCESS",
"created":"2019-01-29T16:28:59+0000"
}
```

I added `id` to the manifest history API, it looks like we can return a uuid too
```javascript
{
"id":"4028f92a689a347f01689a7140c002b7",
"statusMessage":"test_manifest_roles file imported successfully.",
"status":"SUCCESS",
"created":"2019-01-29T16:28:59+0000"}
```

I can just use the id from the API after adding it there, I don't see a reason to not add it in the API. I'll revert the table changes too since I think we want to continue to use `id` until there is a reason not too